packages/commonwealth/client/scripts/helpers/formValidationMessages.ts

```diff
@@ -0,0 +1,5 @@
+export const VALIDATION_MESSAGES = {
+  NO_INPUT: 'No input',
+  MAX_CHAR_LIMIT_REACHED: 'Max character limit reached',
+  INVALID_INPUT: 'Invalid input',
```

Comment:
not sure about this file
1. do we even need this, considering, that we should reuse schema from https://github.com/hicommonwealth/commonwealth/pull/5800
2. If we need it still, it should be probably closer to the `BasicInformationStep` directory as it would be specific for our current needs

Reply:
1. not sure, replied to a similar question [here](https://github.com/hicommonwealth/commonwealth/pull/5869#discussion_r1420211781)
2. I disagree actually, we are using the same error messages in the gating form so I moved it up to helpers (so we can later on remove it from [here](https://github.com/hicommonwealth/commonwealth/blob/0c57bda5b0b06cb654d6f5a5f50f6ba7fd81ed07/packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/validations.ts#L3-L7) as well)

Reply:
Bump @masvelio

Reply:
@mzparacha 
1. yeah, this is related, let's resolve this comment and keep the discussion there
2. that's fine, didn't know this